modules/rational-defaults.el

```diff
@@ -11,14 +11,44 @@
 
 ;;; Code:
 
+;; Defining variable for XDG directories specifications
+(defvar xdg-data-home-directory
+  (if (getenv "XDG_DATA_HOME")
+    (getenv "XDG_DATA_HOME")
+    (expand-file-name ".local/share/")))
+
+(defvar xdg-cache-home-directory
+  (if (getenv "XDG_DATA_HOME")
+    (getenv "XDG_DATA_HOME")
+    (expand-file-name ".cache/")))
+
+;; Keep emacs folder clean
+
+;; Backup file ( init.el~ )
+(customize-set-variable backup-directory-alist `(("." . ,(expand-file-name "emacs/backups/" xdg-data-home-directory))))
```

Comment:
I don't mind the default Emacs behavior for backups and autosaves, so I would personally prefer not to have these. However, I'd probably prefer to use `var/backups` and `var/autosaves` instead (from the `rational-config-var-directory`) of the xdg folders just so everything stays contained in one place.  If there were a `prefer-xdg-paths` var or something as I mentioned in another comment, that could drive this as well.